src/tutorials/first_game_br.md

```diff
@@ -0,0 +1,166 @@
+
+<!--- TODO image of drop zone cylinder --->
+
+### Focus on the Drop Zone
+
+Inside the **Hierarchy**, navigate to the **Gameplay Settings > State Settings > Round Start Skydive > ServerContext** folder. Select the **DropArea** and press ++F++ to focus the scene on the large cylinder mesh.
+
+<!--- TODO image of hierarchy --->
+
+### Scale the Drop Zone
+
+Press ++R++ to enter scale mode. Change the X and Y scale so the Drop Zone is a different size.
+
+<!--- TODO image of larger drop zone --->
+
+## Change the Falling Player Movement
+
+The player has two different falling phases. When the player drops from the plane, it enters a free fall known as the **Skydive** phase. When the player presses ++space++ while skydiving then it will transition to **Parachute** phase.
+
+### Change Skydive Settings
+
+Navigate to **Gameplay Settings > State Settings > Round Start Skydive > ServerContext**. Select the **SkydivePlayerSettings** and open the Properties window. Increase the **Gravity Scale** property to `3.2` and notice how the player will fall down twice as fast with no parachute.
+
+<!--- TODO image of properties window --->
+
+### Change Parachute Settings
+
+In the same **ServerContext** folder as the **SkydivePlayerSettings**, select the **ParachutePlayerSettings**. In the **Properties** window, change the **Ground > Max Speed** property to `2400`. This will increase the max speed moving forward while parachuting.
+
+<!--- TODO image of properties window --->
+
+## Changing the Play Zone
+
+The "Play Zone" is a cylindrical area where the player will *not* receive constant damage. The Play Zone will shrink in size for a certain amount of intervals.
+
+### Change the Play Zone Settings
+
+Navigate to **Gameplay Settings > Zone Settings > Constricting Play Zone** and open the **Properties** window. The object has many **Custom Properties** that will be described below:
+
+| Property Name | Description |
+| --- | --- |
+| Phase Count | The number of phases. The Play Area will shrink in size as the phases increase. It will stop shrinking once the last phase occurs. |
+| ZoneSizeRatio | The ratio for the Play Area to shrink after each phase. For example, `0.7` signifies the area will shrink down to 70% of its current phase size. |
+| StaticTime | The amount of seconds each phase will last before it starts shrinking. |
+| Closing Time | The amount of seconds it takes to shrink before starting the next phase. |
+| BaseDamageRate | The amount of damage per second applied to the players outside the Play Zone during phase 1. |
+| DamageMultiplier | The amount multiplied to BaseDamageRate after each phase. |
+| UseRoundTiming | The toggle to restart the Play Zone after each round. |
+| ActivationDelay | The amount of seconds until the first phase begins. |
+
+### Change the Initial Play Zone Size
+
+1. Navigate to **Gameplay Settings > Zone Settings > Constricting Play Zone > ServerContext**.
```

Comment:
I wonder if there is a more interesting customization to do here. Either that, or a sentence where you could give context to how this would meaningfully change gameplay.